### ISSUE TITLE
Running sm on macOS - cannot be opened because the developer cannot be verified

### ISSUE THREAD

**Issue by gitizenme** [Boscop/ShaderBoi#1]
![image](https://user-images.githubusercontent.com/643504/116285515-072d7880-a743-11eb-84ab-bcb7492a8d4e.png)

![image](https://user-images.githubusercontent.com/643504/116285647-317f3600-a743-11eb-9bdf-6c224a1782b9.png)

**Comment by Boscop** [comment]
Yeah, when I checked with virustotal I also saw that some antivirus softwares warn about it, so I did a little research, and found that often pyinstaller-created executables are falsely detected as virus based on some heuristic.
Which is why I added this to the Readme:

---

Note: If you get a warning from your antivirus software about the file `ShaderBoi/spatialmedia/sm`, it's a [false positive](https://medium.com/@markhank/how-to-stop-your-python-programs-being-seen-as-malware-bfd7eb407a7). (This file was [created by `pyinstaller`](https://github.com/google/spatial-media/tree/master/spatialmedia#building-standalone-gui-application) from a python script to make it work without python, it's purpose is to add spatial metadata to the recorded video files. If your antivirus software removes this file, ShaderBoi still works, but cannot add spatial metadata to the 360° videos that you render with it. You could add the metadata manually though.)

---
I used pyinstaller so that it doesn't require python to be installed (a large part of the target audience is people using Windows).
Btw, `sm` is only needed when you render 360° videos. ShaderBoi works fine without it.
And if you actually render 360° videos, you can also use the python script directly:
https://github.com/google/spatial-media/tree/master/spatialmedia#inject

E.g. `python spatialmedia -i rec.mp4` for non-stereoscopic or `python spatialmedia -i --stereo=top-bottom rec.mp4` for stereoscopic. (And if it also contains ambisonic audio, add `--spatial-audio`.)

**Comment by Boscop** [comment]
What I could do is: For the linux and mac builds, include the python script (at `spatialmedia/sm.py`), and assume that `python` executable is installed on the user's machine and then invoke that. And only use pyinstaller for the windows build. 
What do you think about this solution? :)
